src/Vts/MonteCarlo/DataStructuresValidation/SimulationInputValidation.cs

```diff
@@ -166,24 +171,25 @@ private static ValidationResult ValidateCombinedInputParameters(SimulationInput
                         {
                             case true when
                                 ellipsoid.Center.X != 0.0 && ellipsoid.Center.Y != 0.0:
-                                return new ValidationResult(
-                                    false,
-                                    "Ellipsoid must be centered at (x,y)=(0,0) for cylindrical tallies",
-                                    "Change ellipsoid center to (0,0) or specify non-cylindrical type tally");
+                                    Console.WriteLine("Warning: off center ellipsoid in tissue with cylindrical detector defined: user discretion advised");
+                                    return new ValidationResult(
+                                    true,
+                                    "Warning: off center ellipsoid in tissue with cylindrical detector defined",
+                                    "User discretion advised: change ellipsoid center to (0,0) or specify non-cylindrical type tally");
                             case true when ellipsoid.Dx != ellipsoid.Dy:

```

Comment:
Same comparison issue as above:
https://www.jetbrains.com/help/resharper/CompareOfFloatsByEqualityOperator.html

Reply:
@lmalenfant, thanks for your review.  I can work on those.

Reply:
Here I agree (although still user input so shouldn't different), so have modified code.